assembly/src/assembler/mod.rs

```diff
@@ -496,21 +526,22 @@ impl Assembler {
                     )
                     .with_span(proc_alias.span());
 
-                    let proc_mast_root = self.resolve_target(
+                    let proc_node_id = self.resolve_target(
                         InvokeKind::ProcRef,
                         &proc_alias.target().into(),
                         &pctx,
                         mast_forest_builder,
                     )?;
+                    let proc_mast_root =
+                        mast_forest_builder.get_mast_node(proc_node_id).unwrap().digest();
 
                     // insert external node into the MAST forest for this procedure; if a procedure
-                    // with the same MAST rood had been previously added to the builder, this will
+                    // with the same MAST root had been previously added to the builder, this will
                     // have no effect
```

Comment:
I think this comment should go above line 529 - no?

Reply:
I actually removed this comment (in #1466) since it's now redundant with the `Assembler::resolve_target()` docs